services/static-webserver/client/source/class/osparc/desktop/credits/CreditsNavBarContainer.js

```diff
@@ -0,0 +1,84 @@
+/* ************************************************************************
+
+   osparc - the simcore frontend
+
+   https://osparc.io
+
+   Copyright:
+     2024 IT'IS Foundation, https://itis.swiss
+
+   License:
+     MIT: https://opensource.org/licenses/MIT
+
+   Authors:
+     * Odei Maiz (odeimaiz)
+
+************************************************************************ */
+
+qx.Class.define("osparc.desktop.credits.CreditsNavBarContainer", {
+  extend: qx.ui.core.Widget,
+
+  construct: function() {
+    this.base(arguments);
+
+    this._setLayout(new qx.ui.layout.Grow());
+
+    this.set({
+      zIndex: osparc.utils.Utils.FLOATING_Z_INDEX,
+      maxWidth: this.self().WIDTH,
+      backgroundColor: "background-main-2",
```

Comment:
I think if we are going to use these we need to define the use-case for each as this name means very little which is why I starting to move more and more away. I think also the color should also be the same as our other dropdown menu for consistency?

Reply:
Implemented 👍